vizro-core/src/vizro/static/css/figures.css

```diff
@@ -39,7 +39,7 @@ Note the relative url works with both serve_locally=True and serve_locally=False
 /* This should be replaced, when the bootstrap theme is finalized. Currently our font tokens don't match with the
  bootstrap font tokens, leading to deviations which we need to overwrite. */
 .card-kpi-title {
-  font-size: 1.5rem;
+  font-size: 1.2rem;
```

Comment:
I've reduced the title for all KPI cards, as they previously had the same font-size as the page-title, which was too big